app/src/main/java/com/example/trempelapp/presentation_layer/splash_screen/TrempelSplashActivity.kt

```diff
@@ -20,8 +23,25 @@ class TrempelSplashActivity : BaseActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         (application as TrempelApplication).trempelApp.inject(this)
+        trempelSplashViewModel.injectDagger(application as TrempelApplication)
```

Comment:
you inject nothing here. I think it's redundant

Reply:
done